src/pages/instances/InstanceList.tsx

```diff
@@ -73,12 +74,17 @@ const InstanceList: FC = () => {
   const { project } = useParams<{ project: string }>();
   const [createButtonLabel, _setCreateButtonLabel] =
     useState<string>("Create instance");
-  const [filters, setFilters] = useState<InstanceFilters>({
-    queries: [],
-    statuses: [],
-    types: [],
-    profileQueries: [],
-  });
+  // const location = useLocation();
+  const [searchParams] = useSearchParams();
+
+  const filters = {
+    queries: searchParams.getAll("queries"),
+    statuses: enrichStatuses(
+      searchParams.getAll("statuses") as LxdInstanceStatus[],
+    ),
+    types: searchParams.getAll("types"),
+    profileQueries: searchParams.getAll("profileQueries"),
+  };
```

Comment:
Now that these have become query parameters, they can probably get a more concise name: `queries` can become `q` or `query`, `statuses` can become `status` (we have one query parameter for each status that we want to filter, so it doesn't need the plural), `types` can be `type` and `profileQueries` can be `profile`. Also, we might want to explicitly type this object. So, in short, this can become:
```suggestion
  const filters: InstanceFilters = {
    queries: searchParams.getAll("q"),
    statuses: enrichStatuses(
      searchParams.getAll("status") as LxdInstanceStatus[],
    ),
    types: searchParams.getAll("type"),
    profileQueries: searchParams.getAll("profile"),
  };
```